src/mono/wasm/runtime/startup.ts

```diff
@@ -302,6 +384,11 @@ export function bindings_lazy_init(): void {
 
 // Initializes the runtime and loads assemblies, debug information, and other files.
 export async function mono_load_runtime_and_bcl_args(config: MonoConfig | MonoConfigError | undefined): Promise<void> {
```

Comment:
Is this method still needed?

Reply:
It was on `MONO` API for some time now ... not sure about it.